storage/src/main/java/com/tencent/rss/storage/handler/impl/LocalFileReplicaReadHandler.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.storage.handler.impl;
+
+import com.tencent.rss.common.exception.RssException;
+import org.roaringbitmap.longlong.Roaring64NavigableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.tencent.rss.client.api.ShuffleServerClient;
+import com.tencent.rss.client.request.RssGetShuffleDataRequest;
+import com.tencent.rss.client.request.RssGetShuffleIndexRequest;
+import com.tencent.rss.client.response.RssGetShuffleDataResponse;
+import com.tencent.rss.common.ShuffleDataResult;
+import com.tencent.rss.common.ShuffleDataSegment;
+import com.tencent.rss.common.ShuffleIndexResult;
+
+public class LocalFileReplicaReadHandler extends SkippableReadImpl {
```

Comment:
it's better to be LocalFileClientReadHandler

Reply:
LocalFileClientReadHandler can be refactored to LocalFileQuorumReadHandler for all replicas.

LocalFileReplicaReadHandler for each replica.